tools/NewModuleScript/src/main/java/ScriptMain.java

```diff
@@ -143,7 +143,7 @@ private void exitProgram(Exception e){
 
```

Comment:
NewModuleScript has a `build.gradle` too. We should convert that to `build.gradle.kts` as well ?